File: 21_09_2022/520._Detect_Capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        if(word.islower()):
            return True
        if(word.isupper()):
            return True
        else:
            if(word[0].isupper() and word[1:].islower()):
                return True
        
        return False

File: 21_09_2022/520._Detect_Capital.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int upp=0,low=0;
        for(auto ch:word){
            if(isupper(ch)){
                upp++;                
            }
            else{
                low++;
            }
        }
        int n=word.length();
        if(upp==n){
            return true;
        }
        else if(low==n){
            return true;
        }
        else if(isupper(word[0]) && low>=1 && upp==1){
            return true;
        }
        else{
            return false;
        }
    }
};